java/org/prism/Parser.java

```diff
@@ -1,11 +1,96 @@
 package org.prism;
 
+import java.io.ByteArrayOutputStream;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+// @formatter:off
 public class Parser {
 
+    public static class ParsingOptions {
```

Comment:
@andrykonchin Could you move this to a separate file?
It seems better because (I think) the Parser class around only class-loads successfully if there is the corresponding JNI method implemented.
If it's in another file that's no longer a concern.

Reply:
Done